storybook/src/Heba/atoms/ManImage/Man.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Man1 from "../../images/image-41.png";
+import Man2 from "../../images/image-42.png";
+import Man3 from "../../images/image-43.png";
+import Man4 from "../../images/image-44.png";
+
+export const MAN_OPTION = {
+  TYPE: {
+    Man1: "Man1",
+    Man2: "Man2",
+    Man3: "Man3",
+    Man4: "Man4",
+  },
+};
+
+export const Man = ({ type }) => {
```

Comment:
buttons appear to be mispositioned on photos.
![image](https://user-images.githubusercontent.com/44810632/107155112-4687cc80-697f-11eb-827c-428db708346f.png)

Reply:
the same 
oooops
i found the issue ):
your screen layout in almost different from my screen this shape is justify on my desktop screen

Reply:
let me know your screen setup/resolution

Reply:
![New Bitmap Image](https://user-images.githubusercontent.com/68795300/107866640-fed3da00-6e83-11eb-948f-21266570a4a6.jpg)